timemachine/cpp/src/nonbonded_interaction_group.cu

```diff
@@ -370,29 +375,30 @@ void NonbondedInteractionGroup<RealType>::set_atom_idxs_device(
     if (NC + NR != N_) {
         throw std::runtime_error("Total of indices must equal N");
     }
-    const size_t tpb = warp_size;
+    if (NR > 0 && NC > 0) {
+        const size_t tpb = warp_size;
```

Comment:
Don't like this, but either we can add and remove the ixn group to the `all_potentials_` of the LocalMDPotentials class, which seems like more complicated bookkeeping though would avoid a potential being a no-op.

Reply:
These changes are also likely to conflict with https://github.com/proteneer/timemachine/pull/1011